test/serializer/basic/DefinePropertySameNameDiffDesc.js

```diff
@@ -0,0 +1,6 @@
+(function() {
```

Comment:
Due to a bug in node/test-runner, this testcase incorrectly succeeds even without this change. 
To workaround this issue, please add following directives to the top of the testcase:
// Copies of enumerable:2
This will make the testcase fail before but succeeds with your change.